pkg/crc/preflight/preflight_windows.go

```diff
@@ -179,62 +135,13 @@ var vsockChecks = []Check{
 
 var errReboot = errors.New("Please reboot your system and run 'crc setup' to complete the setup process")
 
-var adminHelperGroup = Check{
-	check: func() error {
-		_, _, err := powershell.Execute(fmt.Sprintf("Get-LocalGroupMember -Name crc-users -Member '%s'", username()))
-		return err
-	},
-	fix: func() error {
-		_, _, err := powershell.ExecuteAsAdmin("adding current user to crc-users group", fmt.Sprintf("Add-LocalGroupMember -Group crc-users -Member '%s'", username()))
-		if err != nil {
-			return err
-		}
-		return errReboot
-	},
-}
-
 func username() string {
 	if ok, _ := win32.DomainJoined(); ok {
 		return fmt.Sprintf(`%s\%s`, os.Getenv("USERDOMAIN"), os.Getenv("USERNAME"))
 	}
 	return os.Getenv("USERNAME")
 }
 
-var hypervGroup = Check{
-	check: checkIfUserPartOfHyperVAdmins,
-	fix:   fixUserPartOfHyperVAdmins,
-}
-
-var adminHelperChecks = []Check{
-	{
-		configKeySuffix:  "check-admin-helper-daemon-installed",
-		checkDescription: "Checking if admin-helper daemon is installed",
-		check: func() error {
-			version, err := adminhelper.Client().Version()
-			if err != nil {
-				return err
-			}
-			expected := filepath.Base(constants.DefaultAdminHelperCliBase)
-			if version != expected {
-				return fmt.Errorf("unexpected admin-helper daemon version (expected: %s, got: %s)", expected, version)
-			}
-			return nil
-		},
-		fixDescription: "Installing admin-helper daemon",
-		fix: func() error {
-			_, _, err := powershell.ExecuteAsAdmin("install admin-helper daemon", fmt.Sprintf("& '%s' uninstall-daemon; & '%s' install-daemon", adminhelper.BinPath, adminhelper.BinPath))
-			return err
-		},
-		cleanupDescription: "Uninstalling admin-helper daemon",
-		cleanup: func() error {
-			_, _, err := powershell.ExecuteAsAdmin("uninstall admin-helper daemon", fmt.Sprintf("& '%s' uninstall-daemon", adminhelper.BinPath))
-			return err
-		},
-		labels: labels{Os: Windows},
-	},
-	cleanUpHostsFile,
-}
-
```

Comment:
These admin-helper/hyperv related removals may belong to the previous commit.

Reply:
I kept it in this commit since this related to the admin-helper binary so i kept it here, the previous commit was about the `crc-users` group and hyper-v admins group.